src/components/TextInput.tsx

```diff
@@ -84,14 +87,15 @@ export class CTextInput extends React.Component<Props, State> {
           onFocus={this.handleInputFocus}
           onBlur={this.handleInputBlur}
         />
+        {right}
```

Comment:
i expected the right element to be on the right? shouldn't this come after the circle button then?

Reply:
I was not sure about whether to prioritise the clear button or this custom button. I'll check with Nitya. I'll make the change for now.

In our examples we'll have only one showing at the same time.

| {right} in the middle | {right} on the right-most side |
| -- | -- |
| ![image](https://github.com/valora-inc/wallet/assets/147835134/138136a1-0643-475c-b024-0260c2ca1a00) | ![image](https://github.com/valora-inc/wallet/assets/147835134/85245817-09f3-4ad0-a9aa-f26d6e9aa03e) |